otx/cli/tools/train.py

```diff
@@ -134,17 +153,40 @@ def main():
     dataset_class = get_dataset_class(template.task_type)
 
     # Create instances of Task, ConfigurableParameters and Dataset.
+    train_ann_files, train_data_roots = args.train_ann_files, args.train_data_roots
```

Comment:
how about codes to make data_roots (or dataset?) as separate function?

Reply:
As per your advice, I moved it into a separate function (`otx/cli/utils/config.py::configure_dataset`) and modified it to be used when configuring the `train`, `eval`, and `optimize` datasets in otx cli.